settings.go

```diff
@@ -30,100 +30,171 @@ type SettingsService struct {
+	SnowplowCollectorHostname                 *string           `json:"snowplow_collector_hostname,omitempty"`
+	SnowplowCookieDomain                      *string           `json:"snowplow_cookie_domain,omitempty"`
+	SnowplowEnabled                           *bool             `json:"snowplow_enabled,omitempty"`
+	SnowplowSiteId                            *string           `json:"snowplow_site_id,omitempty"`
+	TerminalMaxSessionTime                    *int              `json:"terminal_max_session_time,omitempty"`
+	Terms                                     *string           `json:"terms,omitempty"`
+	ThrottleAuthenticatedAPIEnabled           *bool             `json:"throttle_authenticated_api_enabled,omitempty"`
+	ThrottleAuthenticatedAPIPeriodInSeconds   *int              `json:"throttle_authenticated_api_period_in_seconds,omitempty"`
+	ThrottleAuthenticatedAPIRequestsPerPeriod *int              `json:"throttle_authenticated_api_requests_per_period,omitempty"`
+	ThrottleAuthenticatedWebEnabled           *bool             `json:"throttle_authenticated_web_enabled,omitempty"`
+	ThrottleAuthenticatedWebPeriodInSeconds   *int              `json:"throttle_authenticated_web_period_in_seconds,omitempty"`
+	ThrottleAuthenticatedWebRequestsPerPeriod *int              `json:"throttle_authenticated_web_requests_per_period,omitempty"`
+	ThrottleUnauthenticatedEnabled            *bool             `json:"throttle_unauthenticated_enabled,omitempty"`
+	ThrottleUnauthenticatedPeriodInSeconds    *int              `json:"throttle_unauthenticated_period_in_seconds,omitempty"`
+	ThrottleUnauthenticatedRequestsPerPeriod  *int              `json:"throttle_unauthenticated_requests_per_period,omitempty"`
+	TimeTrackingLimitToHours                  *bool             `json:"time_tracking_limit_to_hours,omitempty"`
+	TwoFactorGracePeriod                      *int              `json:"two_factor_grace_period,omitempty"`
+	UniqueIPsLimitEnabled                     *bool             `json:"unique_ips_limit_enabled,omitempty"`
+	UniqueIPsLimitPerUser                     *int              `json:"unique_ips_limit_per_user,omitempty"`
+	UniqueIPsLimitTimeWindow                  *int              `json:"unique_ips_limit_time_window,omitempty"`
+	UsagePingEnabled                          *bool             `json:"usage_ping_enabled,omitempty"`
+	UserDefaultExternal                       *bool             `json:"user_default_external,omitempty"`
+	UserDefaultInternalRegex                  *string           `json:"user_default_internal_regex,omitempty"`
+	UserOauthApplications                     *bool             `json:"user_oauth_applications,omitempty"`
+	UserShowAddSSHKeyMessage                  *bool             `json:"user_show_add_ssh_key_message,omitempty"`
+	VersionCheckEnabled                       *bool             `json:"version_check_enabled,omitempty"`
+	WebIDEClientsidePreviewEnabled            *bool             `json:"web_ide_clientside_preview_enabled,omitempty"`
```

Comment:
All these fields should _not_ use pointers and should _not_ have `omitempty` in their tags`. Those are only needed in option structs.

Reply:
Done